yarn.lock

```diff
@@ -7234,11 +7234,6 @@ fn.name@1.x.x:
   resolved "https://registry.yarnpkg.com/fn.name/-/fn.name-1.1.0.tgz#26cad8017967aea8731bc42961d04a3d5988accc"
   integrity sha512-GRnmB5gPyJpAhTQdSZTSp9uaPSvl09KoYcMQtsB9rQoOmzs9dH6ffeccH+Z+cv6P68Hu5bC6JjRh4Ah/mHSNRw==
 
-focus-lock@^0.7.0:
```

Comment:
Changing the `yarn.lock` file without changing `package.json`, and not calling it out in the PR description is concerning.  Do you know what is happening here?

Reply:
@bkjohnson reverted - must have come in via a merge conflict - this was not intentional and I did not see it prior to this morning